QUARKUS-2739.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-2739 - Add support for JDBCStore transaction logs
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-2739
+
+Add support for `JDBC journal transaction logs`.
+
+## Scope of the testing
+
+A database transaction log should records all transactions and the database modifications made by each transaction. 
+If there is any fatal system failure, this log should be enough to recover your database back to a consistent state.
+
+Ensure that Narayana extension deals well with JDBC datasource transaction log and all the pending transactions (due to a crash 
+on Quarkus app) are completed when the application recover the initial state. 
+
+Tests should verify:
+- All supported datasources 
+- JVM and Native mode
+- Openshift integration (We know that Openshift will require some operator or persistent volumn in a none ephimeral node in order to test this feature, but this operator doesn´t exist at this moment. So the coverage is going to assume that all K8s / Openshift configuration is stable and mature, but maybe sometimes we could experiment some flaky test due to the ephimeral persistence unit that is under the hood)
```

Comment:
Openshift => OpenShift

Operator is not planned for the initial release, there needs to be documentation describing the recovery procedure. This documentation needs to be provided by eng team. We need to verify the steps.